lib/provider/default_volume_provider_test.go

```diff
@@ -0,0 +1,185 @@
+/**
+ * Copyright 2020 IBM Corp.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+// Package provider ...
+package provider
+
+import (
+	"testing"
+
+	"github.com/stretchr/testify/assert"
+	"go.uber.org/zap"
+)
+
+var (
+	logger *zap.Logger
+)
+
+func init() {
+	logger, _ = zap.NewDevelopment()
+}
+
+func TestForDefaultVolumeProviderTypeAndName(t *testing.T) {
```

Comment:
I did not see any method with name `ForDefaultVolumeProviderTypeAndName`, Can you separate test for each method independenly

Reply:
I wanted to avoid more functions the intention was code coverage , I understand we can have separate function for each function.